src/connectors/numeric-menu/connectNumericMenu.ts

```diff
@@ -260,15 +283,22 @@ function isRefined(state, attribute, option) {
     );
   }
 
-  return undefined;
+  return false;
 }
 
-function refine(state, attribute, items, facetValue) {
+function refine(
+  state: SearchParameters,
+  attribute: string,
+  facetValue: string
+) {
   let resolvedState = state;
 
-  const refinedOption = JSON.parse(window.decodeURI(facetValue));
+  const refinedOption = JSON.parse((window as any).decodeURI(facetValue));
 
-  const currentRefinements = resolvedState.getNumericRefinements(attribute);
+  // @TODO: why is array / element mixed here & hasRefinements; seems wrong?
+  const currentRefinements = resolvedState.getNumericRefinements(
```

Comment:
getNumericRefinements returns an array (according to typings), but we use it as a single OperatorList, which one is correct?